Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -441,14 +451,21 @@ protected virtual GrpcChannelOptions GetChannelOptions()
         /// Returns whether or not a channel pool can be used if a channel is required. The default behavior is to return
         /// true if and only if no quota project, scopes, credentials or token access method have been specified. Derived classes should
         /// override this property if there are other reasons why the channel pool should not be used.
+        /// We do not enable self-signed Jwts with channel pools until credentials are not part of the key
         /// </summary>
         protected virtual bool CanUseChannelPool =>
             ChannelCredentials == null &&
             CredentialsPath == null &&
             JsonCredentials == null &&
             TokenAccessMethod == null &&
             Scopes == null &&
-            QuotaProject == null;
+            QuotaProject == null &&
+            UseJwtAccessWithScopes == GetChannelPool()._useJwtAccessWithScopes;
+
+        /// <summary>
+        ///  Returns whether or not self-signed JWTs will be used over OAuth tokens when OAuth scopes are explicitly set.
+        /// </summary>
+        protected virtual bool UseJwtAccessWithScopes => false;
```

Comment:
This indicates the *default*, right? Is the plan not to allow users to customize this via another property, at the moment?